Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -127,8 +231,8 @@
 		2C872C7C221C93710054F791 /* Security.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Security.framework; path = System/Library/Frameworks/Security.framework; sourceTree = SDKROOT; };
 		2C872C7E221C93800054F791 /* libc++.tbd */ = {isa = PBXFileReference; lastKnownFileType = "sourcecode.text-based-dylib-definition"; name = "libc++.tbd"; path = "usr/lib/libc++.tbd"; sourceTree = SDKROOT; };
 		2CE97718225EA543001F5E22 /* openssl.bash */ = {isa = PBXFileReference; lastKnownFileType = text.script.sh; path = openssl.bash; sourceTree = "<group>"; };
-		2CE97719225EB368001F5E22 /* libcrypto.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; name = libcrypto.a; path = ../../Binaries/mac/lib/libcrypto.a; sourceTree = "<group>"; };
-		2CE9771A225EB368001F5E22 /* libssl.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; name = libssl.a; path = ../../Binaries/mac/lib/libssl.a; sourceTree = "<group>"; };
+		2CE97719225EB368001F5E22 /* libcrypto.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; name = libcrypto.a; path = openssl/lib/libcrypto.a; sourceTree = BUILT_PRODUCTS_DIR; };
+		2CE9771A225EB368001F5E22 /* libssl.a */ = {isa = PBXFileReference; lastKnownFileType = archive.ar; name = libssl.a; path = openssl/lib/libssl.a; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
Since these rely on a product path instead of a path that has its flavor (mac/ios) built into it, is the build smart enough for the following:
From a clean build, if you build iOS framework, then the Mac framework, will the Mac build job know it needs a different SSL product?

Reply:
Yep! That's the big advantage of this. `BUILT_PRODUCTS_DIR` will be `DerivedData/.../Products/Debug` for Mac targets, `.../Products/Debug-iphoneos` for iPhone device targets, and `.../Products/Debug-iphonesimulator` for iOS simulators.